{{cookiecutter.github_repository}}/docs/graphql/2-users.md

```diff
@@ -0,0 +1,68 @@
+## Current User
+(requires authentication)
+
+__Request__
+```
+query {
+    me {
+        id
+        firstName
+        lastName
+        email
+    }
+}
+```
+
+__Response__
+
+```json
+{
+  "data": {
+    "me": {
+      "id": "629b1e03-53f0-43ef-9a03-17164cf782ac",
+      "first_name": "John",
+      "last_name": "Hawley",
+      "email": "john@localhost.com"
+    }
+  }
+}
+```
+
+
+## All Users
+(requires authentication and superuser privilege)
+
+__Request__
+```
+query {
+    users {
+        id
+        firstName
+        lastName
+        email
+    }
+}
+```
+
+__Response__
+
+```json
+{
+  "data": {
+    "users": [
```

Comment:
Should this response have edges and nodes, since we are following relays?

Reply:
Yes, now updated in PR.